code/addons/a11y/src/components/A11YPanel.test.tsx

```diff
@@ -8,6 +8,9 @@ import { A11YPanel } from './A11YPanel';
 import { EVENTS } from '../constants';
 
 jest.mock('@storybook/manager-api');
+
+global.ResizeObserver = require('resize-observer-polyfill');
```

Comment:
Why is a polyfill required?  ResizeObserver seems well-supported, given that we support Chrome 100+ now.  https://caniuse.com/resizeobserver

Reply:
I guess this test is run in jest and I don't believe jsdom supports resizeobserver

Reply:
bingo